internal/cli/version.go

```diff
@@ -0,0 +1,20 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func newVersionCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:   "version",
+		Short: "Version",
+		Long:  `Display application version`,
+		Run:   runVersionCmd,
+	}
+}
+
+func runVersionCmd(_ *cobra.Command, _ []string) {
+	fmt.Printf("Version: %s", Version)
```

Comment:
I would add also the binary name, as most of the binaries:

```
-> % go version 
go version go1.18.3 linux/amd64
-> % git version
git version 2.38.0
```

```suggestion
	fmt.Printf("stego version %s", Version)
```